Unity-Technologies/Graphics
# Fixed two issues with sky static lighting.

### Purpose of this PR
This PR fixes two issues with the static lighting sky.
- Fixed a flickering issue caused by having an active planar probe in the scene while rendering inspector preview (https://fogbugz.unity3d.com/f/cases/1269557/)
- Fixed an issue where even when set to OnDemand, the sky lighting would still be updated when changing sky parameters (no fogbugz)

---
### Testing status

**Manual Tests**: What did you do?
- Other: 

For the first issue: Tested the repro scene which was trivial. The fix involving disabling reflection for inspector previews, I also checked that regular camera previews in the scene view thumbnail would still display reflections.
For the second issue: Made a simple scene with two spheres, one chrome (to see sky reflection) and one white rough (to see static lighting) and tested the following;
- Set the current sky to OnDemand and changed exposure. Before the fix, the chrome sphere reflection would update, with the fix it stays the same. Tried then switching to "real time" or "onChanged" to verify that reflection would still update correctly.
- Tried setting a sky using for static lighting to OnDemand and tested the same to verify that behavior was consistent.
- Tried setting static sky to none and back to verify it would still update.
